assets/scss/components/stencil/optimizedCheckout/_optimizedCheckout.scss

```diff
@@ -0,0 +1,104 @@
+// =============================================================================
+// Stencil Checkout  - Customize the Optimized Single-Page Checkout experience
+// =============================================================================
+
+// Header
+.checkoutHeader {
+    background-image: url(stencilImage('optimizedCheckout_custom_background_image', 'optimizedCheckout_custom_background_image_size'));
+    background-size: cover;
+    background-color: stencilColor("optimizedCheckout_header_backgroundColor");
+
+    // Desktop Only
+    @include breakpoint("medium") {
+        text-align: stencilString("optimizedCheckout_logo_position");
+    }
+}
+
+// Order Summary, Mobile Order Summary, Modal Order Summary
+.cart,
+.cartDrawer,
+.cart-modal-body,
+.cart-modal-header {
+    background-color: stencilColor("optimizedCheckout_cart_backgroundColor");
+}
+
+// Headings
+h1,
+h2,
+h3,
+h4,
+h5,
+h6 {
+    color: stencilColor("optimizedCheckout_color-headings");
+    font-family: stencilFontFamily("optimizedCheckout_headings_font"), Arial, Helvetica, sans-serif;
+    font-weight: stencilFontWeight("optimizedCheckout_headings_font");
+}
+
+// Body & Main content area
+body {
+    color: stencilColor("optimizedCheckout_color-body");
+    background-color: stencilColor("optimizedCheckout_backgroundColor-body");
+    font-family: stencilFontFamily("optimizedCheckout_body_font"), Arial, Helvetica, sans-serif;
+    font-weight: stencilFontFamily("optimizedCheckout_body_font");
+}
+
+// Links
+$anchor-text-decoration:               none;
+$anchor-text-decoration-hover:         none;
+$anchor-font-color:                    stencilColor("optimizedCheckout_color-textLink");
+$anchor-font-color-hover:              stencilColor("optimizedCheckout_color-textLink--hover");
+
+a {
+    color: $anchor-font-color;
+    text-decoration: $anchor-text-decoration;
+
+    &:hover,
+    &:focus {
+        color: $anchor-font-color-hover;
+
+        @if $anchor-text-decoration-hover != $anchor-text-decoration {
```

Comment:
Is there a reason to use an `if` statement? Can you just apply the style regardless?